phobos/scripts/check_meshes.py

```diff
@@ -98,23 +101,25 @@ def main(args):
 
     if args.output is not None:
         if path.exists(args.output):
-            print("ERROR: Won't overwrite " + args.output)
+            log.error(f"Won't overwrite {args.output}")
             sys.exit(1)
         with open(args.output, "w") as f:
-            f.write(dump_json(report))
-        print("Wrote full output to", args.output)
+            f.write(yaml.safe_dump(report))
+        log.info(f"Wrote full output to {args.output}")
```

Comment:
Here you are reverting again my changes, I think there is something going wrong in your merge. yaml.safe_dump must not be used but instead dump_json